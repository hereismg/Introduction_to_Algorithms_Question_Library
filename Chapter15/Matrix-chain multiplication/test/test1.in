2
3
10 100 5 50
6
30 35 15 5 10 20 25