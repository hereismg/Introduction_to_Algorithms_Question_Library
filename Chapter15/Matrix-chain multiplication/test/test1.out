7500
15125